src/gateway/reference/key-management.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Key and Key Set Management in Kong Gateway
+content_type: reference
+---
+
+Note that this is an OSS feature.
```

Comment:
@jschmid1 Does this mean it's OSS only, or that it's a free feature and anyone can use it regardless of if they are OSS, plus, or enterprise?

Reply:
> or that it's a free feature and anyone can use it regardless of if they are OSS, plus, or enterprise?

this ^^^